packages/react/src/components/TextField/index.tsx

```diff
@@ -72,6 +72,7 @@ export default class TextField extends React.Component<
       requiredText,
       multiline,
       'aria-describedby': ariaDescribedby,
+      children,
```

Comment:
Textfield shouldn't have children and vite was real angry when this was not defined.